qemu/tests/offload_checksum_windows.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from virttest import utils_test
+from virttest import error_context
+from virttest import utils_net
+from virttest.utils_windows import virtio_win
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    The rx/tx offload checksum test for windows
+    1) start vm
+    2) set the tx/rx offload checksum of the netkvm driver to tcp
+    3) restart nic, and run file transfer test
+    4) set the tx/rx offload checksum of the nekvm driver to disable
+    5) restart nic, and run file transfer test again
+
+    param test: the test object
+    param params: the test params
+    param env: test environment
+    """
+
+    def set_offload_checksum_windows(vm, is_tx, checksum_config):
+        """
+        Set the tx or rx offload checksum to certain config, for the first nic
+        on windows.
+
+        param vm: the target vm
+        param is_tx: True for tx setting, False for rx setting
+        param checksum_config: the config for checksum settings, one of 'tcp' or 'disable'
+        """
+        if is_tx:
+            param = "Offload.TxChecksum"
+        else:
+            param = "Offload.RXCS"
+        if checksum_config == "tcp":
+            value = "1"
+        else:
+            value = "0"
```

Comment:
If only 0 and 1,  I suggest use `int(True/False)`, or `int(bool(str/None))`

Reply:
Same reason as above, use single line `if else` to make it simpler.